.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: release
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build package
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel
+        python setup.py sdist bdist_wheel
+
+    - name: Upload artifacts
+      if: ${{ success() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheels
+        path: dist/*.whl
+
+    - name: Set up venv
+      run: |
+        sudo apt-get install python3-venv
+        python -m venv ~/myenv
+        source ~/myenv/bin/activate
+        echo "Virtual env: $VIRTUAL_ENV"
+
+    - name: Install pytest and our package
+      run: |
+        pip install pytest
+        cd ..
```

Comment:
Out of interest - why is this needed?

Reply:
Oh, yeah, I had a comment on that at one point, can add back in.  Without the `cd`, 'pip install' was seeing the `pydicom` folder immediately below and gave some error, something about it already being installed.  But now I wonder if that was without the venv, which I added back in near the end...